packages/wrangler/src/pages/functions/buildWorker.ts

```diff
@@ -164,3 +148,135 @@ export function buildWorker({
 		}
 	);
 }
+
+export type RawOptions = {
+	workerScriptPath: string;
+	outfile: string;
+	directory: string;
+	minify?: boolean;
+	sourcemap?: boolean;
+	watch?: boolean;
+	plugins?: Plugin[];
+	onEnd?: () => void;
+	buildOutputDirectory?: string;
+	nodeCompat?: boolean;
+	local: boolean;
+	betaD1Shims?: string[];
+};
+
+/**
+ * This function bundles a raw `_worker.js` Pages file
```

Comment:
`This function bundles the Advanced Mode _worker.js file ...` if comment below applies